spec/metadata/linux/LinuxUtils_spec.rb

```diff
@@ -64,4 +64,34 @@
       end
     end
   end
+
```

Comment:
It would be a good idea to have tests here that expect the exact output of `merge_openstack_services` and `parse_openstack_container_status`.